_posts/2019-02-07-checking-the-status-of-your-api-submitted-deal.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title:  "Checking the Status of Your API Submitted Deal"
+date:   2019-02-07 4:00:00 -0400
+categories: api
+author: Kyle Chan
+---
+
+*Now that you have submitted your first deal via our [Lead Submission Endpoint](http://tech.forwardfinancing.com/api/2017/03/07/using-our-lead-submissions-api.html),
+you're probably wondering how it's doing.*
+
+## Getting Started
+The only things you'll need are 
+- The **lead ID** you received when you first submitted your deal
+- Your **API key**
```

Comment:
I'm fine calling this an **API key** since you talk about the attribute and how to get one in the "Authentication", and it's not a simple attribute returned by one of our endpoints.